js/DropZone/DropZone.js

```diff
@@ -89,7 +92,7 @@ export default class DropZone {
      */
     addFiles (files, meta = {}) {
         const { valid, text } = this.validator.validate(files, this.files.length, this.size);
-        let processedFiles = [];
+        const processedFiles = [];
```

Comment:
Personal preference, I don't like joined variable definitions (there will be lots of examples of this in the PR), for me joined declarations are more difficult to read and change.